src/parameters.jl

```diff
@@ -0,0 +1,160 @@
+        :L_AI => Float64(50.0),
+        :L_OA => nothing,
+        :C_AO => nothing,
+        :C_AI => nothing,
+        :T_atm_ini => Float64(267.0),
+        :T_oce_ini => Float64(271.0),
+        :T_ice_ini => Float64(270.0),
+        :t_max => Float64(1000.0),
+        :delta_t_cpl => nothing,
+        :delta_t_min => Float64(10.0),
+        :n_t_atm => 50,
+        :n_t_oce => 1,
+        :n_atm => 200,
+        :n_oce => 50,
+        :boundary_mapping => "mean",
+        :sin_field_atm => false,
+        :sin_field_oce => false
+    )
+    physical_values = update_C_AO(physical_values)
+    physical_values[:w_min] = pi / physical_values[:t_max]
+    z_ruAI = Float64(max(1e-3, 0.93e-3 * (1 - physical_values[:a_i]) + 6.05e-3 * exp(-17 * (physical_values[:a_i] - 0.5)^2)))
+    physical_values[:z_ruAI] = z_ruAI
+    physical_values[:C_AI] = physical_values[:kappa]^2 / ((log(physical_values[:z_0numA] / physical_values[:z_ruAI]) - psi(physical_values[:z_0numA] / physical_values[:L_AI], true, physical_values[:L_AI] > 0, false)) * (log(physical_values[:z_0numA] / physical_values[:z_rTAI]) - psi(physical_values[:z_0numA] / physical_values[:L_AI], true, physical_values[:L_AI] > 0, true)))
+    physical_values[:delta_t_cpl] = physical_values[:t_max]
+    return physical_values
+end
+
+function update_physical_values(a_i, physical_values)
+    physical_values[:a_i] = a_i
+    z_ruAI = Float64(max(1e-3, 0.93e-3 * (1 - a_i) + 6.05e-3 * exp(-17 * (a_i - 0.5)^2)))
+    physical_values[:z_ruAI] = z_ruAI
+    physical_values[:C_AI] = physical_values[:kappa]^2 / ((log(physical_values[:z_0numA] / physical_values[:z_ruAI]) - psi(physical_values[:z_0numA] / physical_values[:L_AI], true, physical_values[:L_AI] > 0, false)) * (log(physical_values[:z_0numA] / physical_values[:z_rTAI]) - psi(physical_values[:z_0numA] / physical_values[:L_AI], true, physical_values[:L_AI] > 0, true)))
+    return physical_values
+end
+
+function update_C_AO(physical_values)
+    physical_values[:L_OA] = physical_values[:lambda_u]^2 / (physical_values[:T_atm_ini] * physical_values[:alpha_eos] * physical_values[:lambda_T])
+    physical_values[:C_AO] = physical_values[:kappa]^2 / ((log(physical_values[:z_0numA] / physical_values[:z_ruAO]) - psi(physical_values[:z_0numA] / physical_values[:L_AO], true, physical_values[:L_AO] > 0, false) + physical_values[:lambda_u] * (log(physical_values[:lambda_u] * physical_values[:z_0numO] / (physical_values[:z_ruAO] * physical_values[:mu])) - psi(physical_values[:z_0numO] / physical_values[:L_OA], false, physical_values[:L_OA] > 0, false))) * (log(physical_values[:z_0numA] / physical_values[:z_rTAO]) - psi(physical_values[:z_0numA] / physical_values[:L_AO], true, physical_values[:L_AO] > 0, true) + physical_values[:lambda_T] * (log(physical_values[:lambda_T] * physical_values[:z_0numO] / (physical_values[:z_rTAO] * physical_values[:mu])) - psi(physical_values[:z_0numO] / physical_values[:L_OA], false, physical_values[:L_OA] > 0, true))))
```

Comment:
This line (and also, e.g., l.101) is unreadable. Is there a way to split the computation up?